Libs/Mesh/Mesh.h

```diff
@@ -21,11 +20,14 @@ class Mesh
   /// \param filename
   bool write(const std::string &outFilename);
 
+  /// coverage
```

Comment:
Please emulate Image.h/cpp and move the function comments/descriptions to the .cpp file.